blog/2024-09-16-ratify-1.3.0.mdx

```diff
@@ -0,0 +1,66 @@
+
+### Support of periodic retrieval of keys and certificates stored in a KMS
+
+Users configure keys and certificates through Key Management Providers (KMP) resources for signature verification. Without specifying versions in KMP resources, Ratify fetches the latest versions. Keys or certificates may rotate for security reasons, and automatic rotation is a common practice. When rotated, new versions are created, making the current versions outdated. Before Ratify v1.3.0, users had to manually update KMP resources when versions were rotated or disabled. This could result in verification failures for new versions if updates were not performed promptly. From v1.3.0 onwards, the `refreshInterval` parameter is introduced for KMP resources allowing periodic checks for the latest versions and status validation. Future updates will also support keeping a specified number of previous versions to validate older signatures that may still be in use.
+
+Please note that currently, only the KMP of Azure Key Vault has this support. Let us know if you need support for other KMPs. For more information, see the [Azure Key Vault KMP configuration](https://ratify.dev/docs/next/reference/custom%20resources/key-management-providers#azure-key-vault).
+
+### Improving error messages for artifact verification
+
+Clear and concise error messages help users understand what went wrong and how to fix it, reducing support requests and downtime. For developers, well-written error messages provide specific details that can assist in diagnosing and resolving bugs or system issues. Improving error messages is an ongoing initiative at Ratify project. In this release, we've refined the error messages related to artifact verification as per the [proposal](https://github.com/ratify-project/ratify/blob/v1.3.0/docs/proposals/Error-Messages-Improvements.md).
+
+### Introducing new configuration for Cosign keyless verification
+
+Starting with version 1.2.0, trust policies have been introduced for Cosign signature verification, allowing users to fine tune verification policies. This release enhances the feature by adding support for Cosign keyless verification. For more information, see the [Cosign verifier keyless configuration](https://ratify.dev/docs/next/plugins/verifier/cosign#keyless).
+
+## Other enhancements
+
+Since this release, we have strengthened the security posture of the Ratify project through several key implementations:
+
+**Docker Build Attestations**: We now generate attestations, including provenance and a Software Bill of Materials (SBOM), for Ratify images. This improves transparency in the build process, ensuring better traceability and accountability of image contents.

```

Comment:
suggestion to add link so customer can review verification instructions at https://ratify.dev/docs/next/troubleshoot/security